app/javascript/components/App.js

```diff
@@ -24,11 +26,51 @@ export default class App extends Component {
 
   readPatients = () => {
     fetch("/patients")
-      .then((resp) => resp.json())
+      .then((response) => response.json())
       .then((patients) => this.setState({ patientsArray: patients }))
       .catch((errors) => console.log("Patients errors:", errors));
   };
 
+  createPatient = (newPatient) => {
```

Comment:
For extra learning, you should try to convert these promise chains into async / await.